yubihsm/objects.py

```diff
@@ -373,6 +373,44 @@ def put(
         msg += key_enc + key_mac
         return cls._from_command(session, COMMAND.PUT_AUTHENTICATION_KEY, msg)
 
+    @classmethod
+    def put_asym(
+        cls,
+        session,
+        object_id,
+        label,
+        domains,
+        capabilities,
+        delegated_capabilities,
+        pk_oce,
```

Comment:
To be more consistent, this parameter should be given as a public key object, similar to: https://github.com/Yubico/python-yubihsm/blob/master/yubihsm/objects.py#L424